File: src/main/java/me/rsman/BetterMinecraftCore/CommandKits/PlayerCommands.java
package me.rsman.BetterMinecraftCore.CommandKits;

import co.aikar.commands.BaseCommand;
import co.aikar.commands.CommandIssuer;
import co.aikar.commands.annotation.*;
import co.aikar.commands.annotation.Optional;
import co.aikar.commands.bukkit.contexts.OnlinePlayer;
import me.rsman.BetterMinecraftCore.Managers.Command.CommandManager;
import me.rsman.BetterMinecraftCore.Managers.Command.Lang.MessageKeys;
import me.rsman.BetterMinecraftCore.Managers.PlayerManager;
import org.bukkit.entity.Player;

@CommandAlias("bmc|betterminecraftcore")
@Subcommand("player|p")
public class PlayerCommands extends BaseCommand {

    //all config wiped out, using config for database data is useless.
    private final CommandManager commandManager = CommandManager.get();

    @Subcommand("getAttribute")
    @CommandCompletion("@attribute @players")
    @CommandPermission("hc.player.attribute.get")
    @Description("{@@bmc.command.description.attribute.get}")
    @Syntax("<attr> <player>")
    public void onGetAttr(Player playerSender, @Values("@attribute") String attr, @Optional @Values("@players") OnlinePlayer player){
        Player playerChecked = player != null ? player.getPlayer() : playerSender;
        CommandIssuer issuerSender = commandManager.getCommandIssuer(playerSender);
        issuerSender.sendInfo(MessageKeys.PLAYER_ATTRIBUTE, "{player}", playerChecked.getDisplayName(), "{attr}", attr, "{val}", PlayerManager.getAttributes(playerChecked.getUniqueId().toString()).get(attr)+"");
    }
    @Subcommand("setBaseAttribute")
    @CommandCompletion("@attribute <value> @players")
    @CommandPermission("hc.player.attribute.set")
    @Description("{@@bmc.command.description.attribute.set}")
    @Syntax("<attr> <value> <player>")
    public void onSetBaseAttr(Player playerSender, @Values("@attribute") String attr, Long value, @Optional @Values("@players") OnlinePlayer player){
        Player playerChecked = player != null ? player.getPlayer() : playerSender;
        CommandIssuer issuerSender = commandManager.getCommandIssuer(playerSender);
        PlayerManager.setBaseAttribute(playerChecked.getUniqueId().toString(), attr, value);
        issuerSender.sendInfo(MessageKeys.PLAYER_BASE_ATTRIBUTE_SET, "{player}", playerChecked.getDisplayName(), "{attr}", attr, "{val}", value+"");
    }
}

File: src/main/kotlin/me/rsman/BetterMinecraftCore/CommandKits/PlayerCommands.kt
package me.rsman.BetterMinecraftCore.CommandKits

import co.aikar.commands.BaseCommand
import org.bukkit.entity.Player
import co.aikar.commands.CommandIssuer
import co.aikar.commands.annotation.*
import co.aikar.commands.bukkit.contexts.OnlinePlayer
import me.rsman.BetterMinecraftCore.Managers.Command.CommandManager
import me.rsman.BetterMinecraftCore.Managers.Command.Lang.MessageKeys
import me.rsman.BetterMinecraftCore.Managers.PlayerManager

@CommandAlias("bmc|betterminecraftcore")
@Subcommand("player|p")
class PlayerCommands : BaseCommand() {
    //all config wiped out, using config for database data is useless.
    private val commandManager = CommandManager.get()
    @Subcommand("getAttribute")
    @CommandCompletion("@attribute @players")
    @CommandPermission("hc.player.attribute.get")
    @Description("{@@bmc.command.description.attribute.get}")
    @Syntax("<attr> <player>")
    fun onGetAttr(playerSender: Player?, @Values("@attribute") attr: String?, @Optional @Values("@players") player: OnlinePlayer?) {
        val playerChecked = if (player != null) player.getPlayer() else playerSender!!
        val issuerSender: CommandIssuer = commandManager?.getCommandIssuer(playerSender) ?: return
        issuerSender.sendInfo(MessageKeys.PLAYER_ATTRIBUTE, "{player}", playerChecked.displayName, "{attr}", attr, "{val}", PlayerManager.getAttributes(playerChecked.uniqueId.toString())[attr].toString() + "")
    }

    @Subcommand("setBaseAttribute")
    @CommandCompletion("@attribute <value> @players")
    @CommandPermission("hc.player.attribute.set")
    @Description("{@@bmc.command.description.attribute.set}")
    @Syntax("<attr> <value> <player>")
    fun onSetBaseAttr(playerSender: Player?, @Values("@attribute") attr: String?, value: Long, @Optional @Values("@players") player: OnlinePlayer?) {
        val playerChecked = if (player != null) player.getPlayer() else playerSender!!
        val issuerSender: CommandIssuer = commandManager?.getCommandIssuer(playerSender) ?: return
        PlayerManager.setBaseAttribute(playerChecked.uniqueId.toString(), attr ?: return , value)
        issuerSender.sendInfo(MessageKeys.PLAYER_BASE_ATTRIBUTE_SET, "{player}", playerChecked.displayName, "{attr}", attr, "{val}", value.toString() + "")
    }
}